### PR TITLE
Singleton2

### PR DESCRIPTION
**Pull request by PKEv**
cleaned, continued #310

### CODE REVIEW

**Comment by PKEv** [comment]
#76 #238

**Comment by PKEv** [comment]
@oroulet need freback

File: qtodotxt/app.py [hunk 1]
```diff
@@ -84,16 +87,48 @@ def _createController(args):
...
+        else:
+            f = open(tempFileName, 'w')
+            if args.quickadd is False:
+                f.write("1")
+            if args.quickadd is True:
+                f.write("2")
+            f.flush()
+            f.close()
+            sys.exit(-1)
+
```
Comment (oroulet) [inline qtodotxt/app.py]: Can you move that to its own funtion?

**Comment by oroulet** [comment]
singleton works on linux now but ... now it is on all the time. Event if I set the option to off....

**Pull request merged**